src/rhub/api/policies.py

```diff
@@ -56,15 +56,28 @@ def inner(*args, **kwargs):
     return inner
 
 
-def list_policies(user):
+def list_policies(user, filter_, page=0, limit=DEFAULT_PAGE_LIMIT):
     """
     API endpoint to provide a list of policies
     """
-    policies = db.session.query(model.Policy.id,
-                                model.Policy.name,
-                                model.Policy.department).all()
-    columns = ['id', 'name', 'department']
-    return [dict(zip(columns, policy)) for policy in policies]
+    policies = db.session.query(
+        model.Policy.id,
+        model.Policy.name,
+        model.Policy.department,
+    )
+
+    if 'name' in filter_:
+        policies = policies.filter(model.Policy.name.ilike(filter_['name']))
+
+    if 'department' in filter_:
+        policies = policies.filter(model.Policy.name.ilike(filter_['department']))
```

Comment:
typo: `department` instead of `name`

Reply:
You can use `%` in the filter value, for example `/lab/cluster?filter[name]=%test%` to search for substring, `...=test%` for prefix, etc.